src/util.cpp

```diff
@@ -553,6 +575,11 @@ static bool InterpretNegatedOption(std::string& key, std::string& val)
     return false;
 }
 
+void ArgsManager::SelectConfigNetwork(const std::string& network)
```

Comment:
Would be slightly more efficiecnt to take a normal (non-reference) string argument, and std::move() it below to avoid a copy.